<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 0, 0);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/5102.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_warning">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/5107.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_log">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>查看日志</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>231</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_curtain">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>311</width>
     <height>201</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/5108.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>271</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcd_temp">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>294</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>7</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>270</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>湿度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_hum">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>烟雾</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_smoke">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>光照度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_ill">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Co2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_co2">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>气压</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_air">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>PM2.5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_pm">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>燃气</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_gas">
      <property name="numDigits">
       <number>7</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>84</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>人体</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lab_inf">
      <property name="text">
       <string>无人</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>431</y>
     <width>75</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>安卓状态：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_ad">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>431</y>
     <width>30</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>单控</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_fan">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>风扇开</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_warning">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>报警灯开</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_lamp">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>射灯开</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_rfid">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>460</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>门禁开</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_curtain">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>98</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="text">
    <string>窗帘开</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_dandu">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>单控</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>450</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lamp">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>280</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/5111.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>194</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="radio_infrared">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
      </property>
      <property name="text">
       <string>红外</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>通道号</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QSpinBox" name="sp_infrared"/>
    </item>
   </layout>
  </widget>
  <widget class="QRadioButton" name="radio_mode">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>模式</string>
   </property>
   <attribute name="buttonGroup">
    <string>buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QTimeEdit" name="timeEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>hh:mm:ss</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_mode">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>无模式</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>101</y>
     <width>45</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>传感器</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ld1">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>124</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>温度</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>光照</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>湿度</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>条件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld2">
      <item>
       <property name="text">
        <string>&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>100</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_20">
      <property name="text">
       <string>值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ld3"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_18">
    <item>
     <widget class="QLabel" name="label_24">
      <property name="text">
       <string>器件</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld4">
      <item>
       <property name="text">
        <string>风扇</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>报警灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>射灯</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>门禁</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>窗帘</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>160</y>
     <width>87</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_19">
    <item>
     <widget class="QLabel" name="label_25">
      <property name="text">
       <string>动作</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="ld5">
      <item>
       <property name="text">
        <string>开</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>关</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btn_ld">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>开启</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radio_ld">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>230</y>
     <width>117</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>联动</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_draw">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>260</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>图表</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>btn_warning</zorder>
  <zorder>chk_log</zorder>
  <zorder>btn_curtain</zorder>
  <zorder>textEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>lcd_temp</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>layoutWidget_6</zorder>
  <zorder>layoutWidget_7</zorder>
  <zorder>layoutWidget_8</zorder>
  <zorder>label_12</zorder>
  <zorder>lab_ad</zorder>
  <zorder>chk_fan</zorder>
  <zorder>chk_warning</zorder>
  <zorder>chk_lamp</zorder>
  <zorder>chk_rfid</zorder>
  <zorder>chk_curtain</zorder>
  <zorder>radio_dandu</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>btn_lamp</zorder>
  <zorder>widget</zorder>
  <zorder>radio_mode</zorder>
  <zorder>timeEdit</zorder>
  <zorder>lab_mode</zorder>
  <zorder>label_16</zorder>
  <zorder>ld1</zorder>
  <zorder>layoutWidget_9</zorder>
  <zorder>layoutWidget_10</zorder>
  <zorder>layoutWidget_11</zorder>
  <zorder>layoutWidget_12</zorder>
  <zorder>btn_ld</zorder>
  <zorder>radio_ld</zorder>
  <zorder>btn_draw</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
